mad-dog/src/index.js

```diff
@@ -237,6 +238,17 @@ async function main () {
         await testRunner([test])
         break
       }
+      case 'test-listencount': {
```

Comment:
i do think it would be better to combine these into a single test case but i can see the argument for either one - wdyt?